packages/@interaxyz/mobile/src/public/types.tsx

```diff
@@ -100,6 +102,7 @@ export interface PublicAppConfig<tabScreenConfigs extends TabScreenConfig[] = Ta
         // TODO: refine this as we add more assets (e.g. do we want to group by type? or screens? etc)
         welcomeLogo?: React.ComponentType<any>
         welcomeBackgroundImage?: typeof require
+        onboardingSuccessImage?: ImageSourcePropType
```

Comment:
Just for my own understanding, why are this and `welcomeBackgroundImage` typed differently?

Reply:
They are used by different components which have different props

Reply:
Hmm, that makes sense, though I wonder if (at some point) it's worth using consistent types for these sorts of parameters.

Reply:
I added here. Im sure there will be several things to clean up once we see different use cases / needs for the framework. https://linear.app/valora/issue/RET-1315/clean-up-public-interface-before-we-are-out-of-alpha